pages/_app.tsx

```diff
@@ -5,13 +5,17 @@ import { DefaultSeo } from 'next-seo'
 import { getNextSEOConfig } from '../next-seo.config'
 import Head from 'next/head'
 import getConfig from 'next/config'
+import { getLogger } from '../logging/log-util'
 
 import '../styles/globals.css'
 
 // Create a react-query client
 const queryClient = new QueryClient()
 
 const MyApp = ({ Component, pageProps, router }: AppProps) => {
+  const logger = getLogger('app')
+  logger.debug('logger started')
```

Comment:
Not sure if this was intended, but every time I make a request to the application this line is logged..